src/db/migrations/20250219182508_add_notifications_table.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up(knex) {
+  return knex.schema.createTable("notifications", (table) => {
+    table.increments("notification_id").primary();
+    table.string("label").nullable();
```

Comment:
I believe `nullable` is the default, it's fine to include but I expect you can exclude if you want: https://knexjs.org/guide/schema-builder.html#nullable

Reply:
Personally, since it's such an important property, I'd prefer being explicit. If only because it triggers the relevant question: should these actually all be nullable? Are there notifications that don't have a title, for example?

Reply:
I also don't think every field being `nullable` make sense here. I'm assuming that majority of these fields are actually required no?